app/models/access_request.rb

```diff
@@ -25,12 +27,11 @@ def deny
       Mailboxer::Receipt.where(notification: notification).update_all(is_read: true)
     end
   end
-
-  def requested_text
-    map.name + ' - request to edit'
-  end
-
-  def approved_text
-    map.name + ' - access approved'
+  
+  protected
+  
+  def after_created_async
+    NotificationService.access_request(self)
   end
+  handle_asynchronously :after_created_async
```

Comment:
yeah, i was implementing a new standard everywhere else, so I wanted to modify this case to match that